lca-chimevc-stack/SettingAgentId.md

```diff
@@ -5,15 +5,20 @@
 LCA v0.5.0 and later offers the option to identify an agent for a call.   
 
 You provide your own custom logic for retrieving the agent identifier from your contact center / IVR platform.  
-Either:
-1. Assign agent ID when the call starts, OR
-2. Assign agent ID after the call has started
+You have several options:
+1. Assign agent ID when the call streaming starts.
+2. Assign agent ID when you send a START_CALL_PROCESSING event for a previously suspended call.
+2. Assign agent ID at any time after the call has started processing.
 
 ## Assign agent ID when call starts
 
 If known, the AgentID can be provided when the call is started using a ChimeVC CallTranscriber Lambda Hook function - see [./LambdaHookFunction.md](./LambdaHookFunction.md). Your Lambda hook function might retrieve the Agent identifier from the SIPREC invite headers, or (more likely) it might interact with your IVR or some other external system that associates agent with CallId. 
 
-## Assign agent ID after the call has started
+## Assign agent ID when you send a START_CALL_PROCESSING event
+
+If you used a [ChimeVC CallTranscriber Lambda Hook function](./LambdaHookFunction.md) to disable immediate processing of a new call received from ChimeVC, then you can provide the AgentID later if/when yoy send a `START_CALL_PROCESSING` event for that call. This is useful when the call SIPREC stream must be initiated before the caller and agent are connected. See [StartCallProcessingEvent.md](./StartCallProcessingEvent.md) for more information. 
```

Comment:
"disable" immediate processing -> "suspend" immediate processing? Other parts of the docs refer to it as suspend.